app/controllers/comments_controller.rb

```diff
@@ -221,6 +221,13 @@ def hide
     authorize @comment
     @comment.hidden_by_commentable_user = true
     @comment&.commentable&.update_column(:any_comments_hidden, true)
+
+    @comment_notification = Notification.find_by(notifiable_type: "Comment",
```

Comment:
This probably doesn't need to be an instance variable, right? We can make it a local variable or just get rid of it completely:


```suggestion
Notification.find_by(
  notifiable_type: "Comment",
  Notification.find_by(notifiable_type: "Comment",
  notifiable_id: params[:comment_id]
)&.destroy
```